beacon_node/src/config.rs

```diff
@@ -156,6 +156,10 @@ pub fn get_config<E: EthSpec>(
             cli_args.is_present("light-client-server");
     }
 
+    if cli_args.is_present("light-client-server") {
+        client_config.chain.enable_lightclient_server = true;
```

Comment:
Please add a `CommandLineTest` to cover this, similar to:
https://github.com/sigp/lighthouse/blob/c8b2324880ef3d3dd5f9233285cb59fc3701db71/lighthouse/tests/beacon_node.rs#L2364-L2374

Reply:
Added

https://github.com/sigp/lighthouse/blob/6dc45c4d8883d70f7f83c7f1601cdd7fa348a33a/lighthouse/tests/beacon_node.rs#L2355-L2364